lib/compress/zstd_ldm.c

```diff
@@ -466,26 +468,33 @@ static void ZSTD_ldm_reduceTable(ldmEntry_t* const table, U32 const size,
 }
 
 size_t ZSTD_ldm_generateSequences(
-        ldmState_t* ldmState, rawSeq* sequences,
-        ldmParams_t const* params, void const* src, size_t srcSize,
-        int const extDict)
+        ldmState_t* ldmState, rawSeq* sequences, size_t nbSeq,
```

Comment:
`nbSeq` seems to be actually the size (or capacity) of `sequences`.
I believe the name should reflect that.